.circleci/functional_tests_malformed.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+export SHELL=/bin/bash
+
+set -e
+
+echo "Running functional tests..."
+
+#sudo su
+
+# Start the plugin
+sudo /home/circleci/.go_workspace/src/repo/splunk-logging-plugin/rootfs/bin/splunk-logging-plugin &
+rm -rf /opt/circleci/.pyenv
+echo "Creating virtual env to run functional tests..."
+cd test
+curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer | bash
+sudo apt-get update
+sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev git
+export PATH="~/.pyenv/bin:$PATH"
+pyenv install 3.7.0
+pyenv global 3.7.0
+
+
+sudo pip install --upgrade pip
+sudo -H pip install virtualenv
+virtualenv --python=python3.5 venv
+source venv/bin/activate
+pip install -r requirements.txt
+deactivate
+
+#Run functional tests from within virtualenv
+sudo venv/bin/python -m pytest --verbose --cache-clear \
```

Comment:
Can we just iterate through the list (different tests) in a loop? Using For loop for eg

Reply:
done